index.bs

```diff
@@ -397,6 +397,30 @@ changes for most users. Some user agents have taken steps to reduce the
 entropy introduced by [disallowing direct enumeration of the plugin list](https://bugzilla.mozilla.org/show_bug.cgi?id=757726).
 </p>
 
+<h3 class=question id=send-to-platform>
+  Does this specification allow an origin to send data to the underlying
+  platform?
+</h3>
+
+If so, what kind of data can be sent?
+
+Platforms differ in how they process data passed into them,
+which may present different risks to users.
+
+<p class=example>
+URLs may or may not be dereferenced by a platform API,
+and if they are dereferenced,
+redirects may or may not be followed.
+If your specification sends URLs to underlying platform APIs,
+the potential harm of *your* API
+may vary depending on
+the behavior of the various underlying platform APIs it's built upon.
+What happens when `file:`, `data:`, or `blob:` URLs
+are passed to the underlying platform API?
+Keep in mind that, even if your API only allows `http:` and `https:` URLs,
```

Comment:
The implication here seems to be that "http[s]:" is safe.  That's not guaranteed either.  The webshare case showed that the position of a requester is what matters.  Sure, "file:" carries more inherent risk, but only in the aggregate; fetching from "https:" is no less problematic if the resource uses ambient authority to authorize access.

Reply:
@martinthomson, is 68130bf any better?

Reply:
Much.  Thanks.